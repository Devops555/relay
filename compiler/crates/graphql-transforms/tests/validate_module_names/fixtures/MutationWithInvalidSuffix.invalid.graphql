#expected-to-throw
mutation MutationWithInvalidSuffixQuery {
    commentCreate(input: null) {
        clientMutationId
    }
}
