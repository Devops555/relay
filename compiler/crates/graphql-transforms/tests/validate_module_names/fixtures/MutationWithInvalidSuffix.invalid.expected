==================================== INPUT ====================================
#expected-to-throw
mutation MutationWithInvalidSuffixQuery {
    commentCreate(input: null) {
        clientMutationId
    }
}
==================================== ERROR ====================================
Mutations in graphql tags must start with the module name ('MutationWithInvalidSuffix') and end with 'Mutation'. Got 'MutationWithInvalidSuffixQuery' instead.:
MutationWithInvalidSuffix.invalid.graphql:2:10:2:40
mutation MutationWithInvalidSuffixQuery {
